js/api-augment/definitions/handles.ts

```diff
@@ -10,12 +10,26 @@ export default {
       ],
       type: "Option<HandleResponse>",
     },
+    getMsaForHandle: {
+      description: "Get msa_id for a given handle",
+      params: [
+        {
+          name: "display_handle",
```

Comment:
Isn't this a "full handle"? I'm still getting confused by these handle terms.

Reply:
Yes this is full_handle, we will cleanup the term `full_handle` instead keep `display_handle` across the code before final merge

Reply:
Added to Misc Todo on main PR and Robert has fixed most of it